packager/VisibilityChecker.cc

```diff
@@ -215,15 +215,6 @@ class PropagateVisibility final {
             // When exporting a field, we also export its parent namespace. This is a bit of a hack, and it would be
             // great to remove this, but this was the behavior of the previous packager implementation.
             exportParentNamespace(ctx, sym.data(ctx)->owner);
-
-            if (sym.data(ctx)->flags.isStaticFieldTypeAlias) {
-                if (auto e = ctx.beginError(send.loc, core::errors::Packager::ExportingTypeAlias)) {
```

Comment:
Do you mind commenting out the ErrorClass?

You can leave the docs though, as it'll be useful for people on older versions.